modules/message-lane/src/lib.rs

```diff
@@ -110,7 +110,7 @@ pub trait Config<I = DefaultInstance>: frame_system::Config {
 	/// Payload type of outbound messages. This payload is dispatched on the bridged chain.
 	type OutboundPayload: Parameter + Size;
 	/// Message fee type of outbound messages. This fee is paid on this chain.
-	type OutboundMessageFee: Default + From<u32> + Parameter + SaturatingAdd + Zero;
+	type OutboundMessageFee: Default + From<u64> + PartialOrd + Parameter + SaturatingAdd + Zero;
```

Comment:
Why are we changing `From<u32>` to `From<u64>`?

Reply:
The payment for message in some benchmarks doesn't fit `u32` :/ As I noted in the description, right now benchmarks are broken and this is a fix